### DISCUSSION TITLE
Set safe area color on iOS when using contentInset: automatic

### DISCUSSION THREAD

**Discussion by vacekj** [ionic-team/capacitor#4023] [Q&A]
Hi,

thank you for taking the time to look at my question. I am using the `contentInset: automatic` property in capacitor.config.json, and it works correctly, but I was wondering, if there is any way to set the color of the safe area behind the status bar and bottom bar, that is currently white? Example pictures below:

Bottom Navigation bar:
![image](https://user-images.githubusercontent.com/10381200/103777122-b62d3380-5030-11eb-89d2-1eb2f5772f3f.png)

Status Bar:
![image](https://user-images.githubusercontent.com/10381200/103777173-c6451300-5030-11eb-8c60-4d9426f90b22.png)

**Comment by GeorgeL9** [comment]
Anyone figure out a solution?
